Plugins/Flow.Launcher.Plugin.Program/Programs/IProgram.cs

```diff
@@ -9,5 +9,6 @@ public interface IProgram
         string UniqueIdentifier { get; set; }
         string Name { get; }
         string Location { get; }
+        bool Enabled { get; }
```

Comment:
I added a property to the generic interface of all program to check whether they are enabled.  Both win32 and uwp who implement this interface has this property.

Reply:
this is for where user decides to disable the program via the Program plugin context menu or settings right?

Reply:
Yes, and it is also for when program is not loaded correctly. This change won't change anything since the property is already there for win32 and uwp.